runtime/src/iree/builtins/ukernel/query_tile_sizes_internal.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_BUILTINS_UKERNEL_QUERY_TILE_SIZES_INTERNAL_H_
+#define IREE_BUILTINS_UKERNEL_QUERY_TILE_SIZES_INTERNAL_H_
+
+#include "iree/builtins/ukernel/query_tile_sizes.h"
+
+#ifdef __cplusplus
```

Comment:
you shouldn't need the cplusplus guards in internal headers, just the public ones